src/Framework/TestSuite.php

```diff
@@ -78,11 +78,14 @@ class TestSuite implements \IteratorAggregate, SelfDescribing, Test
     protected $foundClasses = [];
 
     /**
-     * Last count of tests in this suite.
-     *
-     * @var null|int
+     * @var null|array<string>
```

Comment:
Not sure, but perhaps this makes sense?

```suggestion
     * @var null|array<callable-string>
```

Reply:
I did not know that existed. :) Much more descriptive, thx @localheinz